dashboard/app/assets/stylesheets/application.css.scss

```diff
@@ -358,10 +358,14 @@ $small-footer-standard-width: 400px;
   position: absolute;
   bottom: 0;
   left: 0;
-  width: 625px;
+  width: 650px;
```

Comment:
It looks like this is already on one line in your before? Or am I missing something?